cmd/influxd/config.go

```diff
@@ -251,6 +246,38 @@ func ParseConfig(s string) (*Config, error) {
 	return c, nil
 }
 
+type Graphite struct {
+	Address       string `toml:"address"`
+	Database      string `toml:"database"`
+	Enabled       bool   `toml:"enabled"`
+	Port          uint16 `toml:"port"`
+	Protocol      string `toml:"protocol"`
+	NamePosition  string `toml:"name-position"`
+	NameSeparator string `toml:"name-separator"`
+}
+
+// Default carbon port per http://graphite.readthedocs.org/en/1.0/feeding-carbon.html
+const defaultGrahitePort = 2004
```

Comment:
I downloaded the graphite source and it looks like `2003` is the default UDP port and `2004` is the default "Pickle" port.  UDP uses the plaintext protocol and Pickle is a more efficient protocol.

http://graphite.readthedocs.org/en/1.0/feeding-carbon.html